Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[workspace]
+members = [
+    "jruby_executable",
+    "ruby_executable",
+    "shared",
+]
+
+[workspace.dependencies]
+java-properties = "2.0.0"
+glob = "0.3.1"
+clap = { version = "4.5.4", features = ["derive"] }
+shared = { path = "shared" }
+indoc = "2.0.5"
+lazy_static = "1.4.0"
```

Comment:
you could drop this dependency for [OnceCell](https://doc.rust-lang.org/std/cell/struct.OnceCell.html) which is in `std`

Reply:
We weren't even using it. Weird that nothing errored or warned from Rust.

Reply:
That's because none of the optional lints are enabled, eg:
https://github.com/heroku/buildpacks-ruby/blob/230e8ddb51a6e6ab60d34287a455017a74659ef7/Cargo.toml#L9-L17
https://github.com/heroku/buildpacks-ruby/blob/230e8ddb51a6e6ab60d34287a455017a74659ef7/clippy.toml